cogs/utility.py

```diff
@@ -10,17 +10,103 @@
 EMOTE_IS_ANIMATED_REGEX = r'(<a)'
 EMOTE_BASE_LINK = 'https://cdn.discordapp.com/emojis/'
 
-with open('assets/convert.json', 'r', encoding='utf-8') as file:
-    raw_convert_table = file.read()
-    convert_table: typing.Dict[object, object] = json.loads(raw_convert_table)
-
-
 def cvt_units(unit1: str, unit2: str, value: float):
-    # Convert temperature units
+    convert_dictionary = {
```

Comment:
I would store this outside of the function so it doesn't have to be created every time it is called.
But then again, it might save memory to not store it outside.

Reply:
I would definitely store it in a separate file and pull it in as function to clean up this file